Source/WTF/wtf/JSONValues.cpp

```diff
@@ -807,7 +872,7 @@ Ref<Array> Array::create()
     return adoptRef(*new Array);
 }
 
-size_t ArrayBase::memoryCost() const
+size_t ArrayBase::memoryCostImpl() const
 {
     size_t memoryCost = Value::memoryCost();
```

Comment:
This needs to be changed to `memoryCostImpl` to avoid infinite recursion.

Reply:
Oops, right. Fixed.